src/lib/components/ui/Modal/Modal.jsx

```diff
@@ -297,29 +281,10 @@ Modal.propTypes = {
    */
   position: PropTypes.oneOf(['top', 'center']),
   /**
-   * How to treat the modal when its content is too long to fit the screen. The `body` option
-   * turns on scrolling just for the modal content, while the `modal` option enables scrolling of
-   * the entire modal, including the header and the footer.
-   */
-  scrollMode: PropTypes.oneOf(['body', 'modal']),
-  /**
-   * Custom CSS for the end scrolling shadow.
-   */
-  scrollViewEndShadowStyle: PropTypes.shape({
-    background: PropTypes.string,
-    boxShadow: PropTypes.string,
-  }),
-  /**
-   * Size of the scrolling shadows.
-   */
-  scrollViewShadowSize: PropTypes.string,
-  /**
-   * Custom CSS for the start scrolling shadow.
+   * The React element to be used as a wrapper of the content providing body scrolling functionality.
+   * If set to `null` the entire modal including header and footer will be scrolled.
    */
-  scrollViewStartShadowStyle: PropTypes.shape({
-    background: PropTypes.string,
-    boxShadow: PropTypes.string,
-  }),
+  scroller: PropTypes.node,
```

Comment:
I would suggest to try something like https://www.npmjs.com/package/airbnb-prop-types and validate that only `ScrollView` can be passed to `scroller`.

Reply:
This is covered by #156  - I added this specific ase to the description there to make sure it does not get forgotten.